server/command.go

```diff
@@ -160,7 +160,12 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 				txt = "### Subscriptions in this channel\n"
 			}
 			for _, sub := range subs {
-				txt += fmt.Sprintf("* `%s` - %s\n", strings.Trim(sub.Repository, "/"), sub.Features)
+				subFlags := fmt.Sprintf("%s", sub.Flags.String())
+				txt += fmt.Sprintf("* `%s` - %s", strings.Trim(sub.Repository, "/"), sub.Features)
+				if subFlags != "" {
+					txt += fmt.Sprintf(" - %s", subFlags)
```

Comment:
I propose removing the single dash prefixing `subFlags`.  See below.  @hanzei, do you agree?

![image](https://user-images.githubusercontent.com/7575921/75697807-377aa380-5c73-11ea-9ac7-d6727d893f23.png)

Reply:
Done.